lib/Yield/Doxygen.cpp

```diff
@@ -0,0 +1,223 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+/// \file Doxygen.cpp
```

Comment:
The whole comment thing is not specific to `Yield`, we're putting way too much stuff in `Yield`.

Let's move this to `librevngPTML`, since it already depends on `librevngModel`.

Reply:
It starts to feel like `librevngPTML` is starting to become what `librevngYield` was intended to be, and I feel responsible because of putting to much stuff in `yield` temporarily 😅.

This comment of yours makes me think that we need to deal with [this thing](https://mattermost.rev.ng/revng/pl/qjsnb8z73if85kfpb7z447cety) sooner that I expected.

Reply:
Yeah, but in the meantime, let's move things there.

Reply:
Done.